.maintain/playbooks/dali-stage.yml

```diff
@@ -109,6 +109,13 @@
         job="certbot certonly --nginx --non-interactive -d '{{ domain }}' --deploy-hook 'nginx -s reload'"
       become: yes
 
+    - name: Lease period prolongation cronjob
+      cron: special_time=hourly
+        name=lease-period-prolongator
+        user=root
+        job="docker run --rm -i -u$(id -u):$(id -g) -eNODE_URL=wss://{{ domain }}/relaychain andor/lease-period-prolongator:0.1.0"
```

Comment:
you may publish via ci to compisable nice tool for others:)

Reply:
I'll do it.